### ISSUE TITLE
Put *tobac* on PyPI

### ISSUE THREAD

**Issue by freemansw1** [tobac-project/tobac#192]
We should probably put *tobac* on PyPI. This isn't urgent and it does make our release process more cumbersome, but it's important for non-conda users. The tutorial is here: https://packaging.python.org/en/latest/tutorials/packaging-projects/

**Comment by JuliaKukulies** [comment]
Good idea @freemansw1 and should probably not cause any problems given that all dependencies in `requirements.txt` exist on PyPI and given that we do not have any external dependencies. 

I am not sure how much more cumbersome this will make our release process though. I have no experience with it but since there are many packages (e.g. `MetPy`) that are published on both the Anaconda cloud and PyPI, we can double check how they do it. I understand from [this blog post](https://levelup.gitconnected.com/publishing-your-python-package-on-conda-and-conda-forge-309a405740cf) that one can specify the location on PyPI as a source in the conda recipe, so that the package on conda-forge is automatically updated when the package is updated on PyPI. Not sure if it also works the other way around?
